app/easyaccess.py

```diff
@@ -150,15 +150,54 @@ def main():
 
     # CONVERT TO ACCESS AND WEB FORMATS
     try:
-        logging.info("Converting to access format...")
-        access_metadata = convert_and_get_metadata(source_file_path, access_file_path, settings.ACCESS_FFMPEG_ARGS, vernon_id, access_file_type, title)
-        logging.info("Converting to access format... DONE\n")
-        if settings.TRANSCODE_WEB_COPY:
-            logging.info("Converting to web format...")
-            web_metadata = convert_and_get_metadata(source_file_path, web_file_path, settings.WEB_FFMPEG_ARGS, vernon_id, web_file_type, title)
-            logging.info("Converting to web format... DONE\n")
+        if settings.EXHIBITIONS_TRANSCODER:
+            # Transcoder settings for in-gallery exhibitions videos
+            logging.info('Converting to exhibitions access format...')
+            access_metadata = convert_and_get_metadata(
+                source_file_path,
+                access_file_path,
+                settings.EXHIBITIONS_ACCESS_FFMPEG_ARGS,
+                vernon_id,
+                access_file_type,
+                title,
+            )
+            logging.info('Converting to access format... DONE\n')
+            if settings.TRANSCODE_WEB_COPY:
+                logging.info('Converting to web format...')
+                web_metadata = convert_and_get_metadata(
+                    source_file_path,
+                    web_file_path,
+                    settings.EXHIBITIONS_WEB_FFMPEG_ARGS,
+                    vernon_id,
+                    web_file_type,
+                    title,
+                )
+                logging.info('Converting to web format... DONE\n')
+        else:
+            # Transcoder settings for collections videos
+            logging.info('Converting to access format...')
+            access_metadata = convert_and_get_metadata(
+                source_file_path,
+                access_file_path,
+                settings.ACCESS_FFMPEG_ARGS,
+                vernon_id,
+                access_file_type,
+                title,
+            )
+            logging.info('Converting to access format... DONE\n')
+            if settings.TRANSCODE_WEB_COPY:
+                logging.info('Converting to web format...')
+                web_metadata = convert_and_get_metadata(
+                    source_file_path,
+                    web_file_path,
+                    settings.WEB_FFMPEG_ARGS,
+                    vernon_id,
+                    web_file_type,
+                    title,
+                )
+                logging.info('Converting to web format... DONE\n')
     except Exception as e:
-        return post_slack_exception("Could not convert to access and web formats: %s" % e)
+        return post_slack_exception('Could not convert to access and web formats: %s' % e)
 
```

Comment:
What do you think about extracting our exhibitions & collector conversions to their own methods to make this a bit easier to read?

```
# CONVERT TO ACCESS AND WEB FORMATS
    try:
        if settings.EXHIBITIONS_TRANSCODER:
            # Transcoder settings for in-gallery exhibitions videos
            access_metadata, web_metadata = convert_to_exhibition_formats(access_file_path, access_file_type,
                                                                          source_file_path, title, vernon_id,
                                                                          web_file_path, web_file_type)
        else:
            # Transcoder settings for collections videos
            access_metadata, web_metadata = convert_collections_formats(access_file_path, access_file_type,
                                                                        source_file_path, title, vernon_id,
                                                                        web_file_path, web_file_type)
    except Exception as e:
        return post_slack_exception('Could not convert to access and web formats: %s' % e)

````